pkgs/top-level/all-packages.nix

```diff
@@ -40728,4 +40728,6 @@ with pkgs;
   reindeer = callPackage ../development/tools/reindeer { };
 
   charasay = callPackage ../tools/misc/charasay { };
+
+  please-cli = callPackage ../applications/misc/please-cli { };
```

Comment:
Please sort this into a fitting category and sort there alphabetically.

Reply:
Do you mean that I should choose something different than `applications/misc`?

Reply:
I basically went down [that](https://nixos.org/manual/nixpkgs/stable/#sec-hierarchy) flow-chart and ended up in `applications/misc`. The program is interactive: You give it a prompt, it returns a shell command and asks if it should run it.
As such, it would also fit into `tools/system`, but tools states `non-interactivity` for this category.

Reply:
I guess we can consider this as resolved.

Reply:
> I guess we can consider this as resolved.

No, this was not resolved.

The directory is fine but new entries should not be appended because if everyone would do that we would have constant merge conflicts.

Reply:
![image](https://github.com/NixOS/nixpkgs/assets/252042/3e4c44e9-6a6e-401a-8275-e4a564f40473)

This looks at the right place to me, what's wrong there?

Reply:
You're right. I am not sure if GitHub threw the information away in the meantime or I had tomatos on my eyes.

Reply:
No problem :)